packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -253,16 +233,13 @@ export default class FetchCache implements CacheHandler {
           throw new Error(`invalid response from cache ${res.status}`)
         }
 
-        const json: IncrementalCacheValue = await res.json()
-        const parsed = zCachedFetchValue.safeParse(json)
+        const cached = await res.json()
```

Comment:
```suggestion
        const cached: IncrementalCacheValue = await res.json()
```
Let's avoid the any here again this was key part I think